.github/workflows/test.yml

```diff
@@ -26,11 +26,24 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
-      
+
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Install unixodbc on AMD
+        if: matrix.architecture == 'amd64'
+        run: |
+          sudo apt install unixodbc
```

Comment:
I think you just need to add this requirement in `Dockerfile*` (regular, release, and cugraph). This is only required if we are to test something

Reply:
@antepusic can you just confirm on this one that we don't need, why did you put that here?